rasa_nlu/training_data/data_loader.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import json
+
+from rasa_nlu import utils
+from rasa_nlu.training_data.formats import MarkdownReader, WitReader, LuisReader
+from rasa_nlu.training_data import TrainingData
+
+logger = logging.getLogger(__name__)
+
+# Different supported file formats and their identifier
+WIT_FILE_FORMAT = "wit"
+DIALOGFLOW_FILE_FORMAT = "dialogflow"
+LUIS_FILE_FORMAT = "luis"
+RASA_FILE_FORMAT = "rasa_nlu"
+UNK_FILE_FORMAT = "unk"
+MARKDOWN_FILE_FORMAT = "md"
+
+
+class DataLoader(object):
```

Comment:
I don't really think we need this class as it really contains no state.

Reply:
👍